src/App.test.js

```diff
@@ -1,8 +1,8 @@
 import { render, screen } from '@testing-library/react';
-import App from './App';
+import Calculator from './Calculator';
 
 test('renders learn react link', () => {
-  render(<App />);
```

Comment:
- [ ] Please your project is missing an `App.js` component, please kindly add that, and make sure is is class-based as [requested](https://github.com/microverseinc/curriculum-react-redux/blob/main/math-magicians/project_components.md) for this project, `Use class based components (All components should be class-based).` " `Once the Calculator is ready, import it and use it in your main component (App.js).`, ". After adding the `App.js` component, then you import the `Calculator.js` component into the `App.js`  component and then the `App.js`  component into the `index.js` file.